spec/requests/swagger_spec.rb

```diff
@@ -193,38 +193,44 @@
       expect(subject).to validate(:get, '/v0/education_benefits_claims/stem_claim_status', 200)
     end
 
-    it 'supports adding an caregiver\'s assistance claim' do
-      VCR.use_cassette 'mpi/find_candidate/valid' do
-        VCR.use_cassette 'emis/get_veteran_status/valid' do
-          VCR.use_cassette 'carma/auth/token/200' do
-            VCR.use_cassette 'carma/submissions/create/201' do
-              VCR.use_cassette 'carma/attachments/upload/201' do
-                expect(subject).to validate(
-                  :post,
-                  '/v0/caregivers_assistance_claims',
-                  200,
-                  '_data' => {
-                    'caregivers_assistance_claim' => {
-                      'form' => build(:caregivers_assistance_claim).form
+    describe 'using salesforce' do
+      before do
+        allow(Flipper).to receive(:enabled?).with(:caregiver_mulesoft).and_return(false)
+      end
+
+      it 'supports adding an caregiver\'s assistance claim' do
+        VCR.use_cassette 'mpi/find_candidate/valid' do
```

Comment:
I really would like to pair at some point to see if there is an easier way to deal this. Sometimes you do need more than one tape in a test but this is a little excessive.